product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/01_private_endpoint.mdx

```diff
@@ -0,0 +1,148 @@
+      Depending on the visibility level you set, the client application might need to request access permission from the EDB Cloud tenant to access this private link
+    service. See [control service exposure](https://docs.microsoft.com/en-us/azure/private-link/private-link-service-overview#control-service-exposure)
+    for details.
+    
+
+  5. After the Private Link Service is created, note down its *alias*. The *alias*
+   is the unique ID for your private service, which can be shared with the service consumers. Obtain the alias either from the Azure portal
+   or by the using the following CLI command:
+
+     ```
+       $ az network private-link-service list --query 
+      "[?name=='pg-service-private-link'].alias" -o tsv
+      pg-service-private-link.48f26b42-45dc-4e80-8e3d-307d58d7d274.japaneast.azure.privatelinkservice
+     
+     ```
+  6. Select **Review + Create**.
+
+  7. Select **Create**.
+
+#### Step 2: Create an Azure Private Endpoint linking to the Private Link Service
+
+  To connect to the PG service, create an Azure Private Endpoint in your VM's virtual network. After you have created the private endpoint, you can use its private IP address to access the PG service.
+  
+  1. On the upper-left side of the screen in the portal, select **Create a resource > Networking > Private Link** or in the search box enter Private Link.
+  
+  1. Select **Create**.
+
+  1. In Private Link Center, select **Private endpoints** in the left-hand menu.
+
+  1. In Private endpoints, click **Create**.
+
+  1. Enter "pg-service-private-link" as the name for the Private Link Service and provide additional details as shown in the following screenshot.
+
+   ![](../images/image12.png)
+
+  1. Connect the private endpoint to the EDB Cloud private link service that we created by entering its *alias*.
+
+   ![](../images/image17.png)
+
+  1. In the **Configuration** page, enter the VM's Virtual Network `vnet-client`.
+
+  1. Select **Review + Create**.
+
+  7. Select **Create**.
+   
+  8. You might have to approve the connection request from the `pg-service-private-link` owner, if the private endpoint's status appears as **Pending** for a long period. 
+   
+   Switch to the EDB Cloud subscription and from the **Pending connections** page in **Private Link Center** approve the connection request.
+
+  ![](../images/image3.png)
+
+
+11. Now that you have successfully built a tunnel between your VM's virtual network and the EDB Cloud cluster network, access the private PG service by using the private endpoint.
+   
+   Get the private endpoint's private IP address using the following scripts:
+
+ ```
+  $ NICID=$(az network private-endpoint show -n pl-private-pg-service
+  -g rg-client --query "networkInterfaces[0].id" -o tsv)
+  $ az network nic show -n ${NICID##*/} -g rg-client --query
+  "ipConfigurations[0].privateIpAddress" -o tsv
+  100.64.111.5
+ ``` 
+
+12. From the VM `vm-client`, access the PG service by using the private IP address.
```

Comment:
I can't agree more with this. TBH, I don't know what I should call it when writing the doc, 
- "PG Service" ?
- "PG Cluster" ?
- "Postgres Service"?
- "Postgres Cluster"?
- "EDB Cloud Cluster"?
- ....

now it's even complex because of rebranding. Should doc team standardize the naming convention here?

Reply:
It should be consistent with the rest of the documentation - I think we usually just use "cluster"? /cc @drothery-edb.

Reply:
let's go with cluster for now. I think we need an internal terminology doc as a complement to our style guide to make sure we are using agreed upon terms consistently. I'll start one and add cluster as our first term along with a definition.